api/src/main/java/org/apache/iceberg/ContentFile.java

```diff
@@ -102,6 +102,18 @@
    */
   List<Long> splitOffsets();
 
+  /**
+   * Returns the set of field IDs used for equality comparison, in equality delete files.
+   * <p>
+   * An equality delete file may contain additional data fields that are not used by equality
+   * comparison. The subset of columns in a delete file to be used in equality comparison are
+   * tracked by ID. Extra columns can be used to reconstruct changes and metrics from extra
```

Comment:
I think there might be a minor typo here - maybe after 'from extra'

Reply:
I think it was intended. The second half of the sentence is "Metrics from extra columns are used during job planning". We might be able to rephrase it to avoid the double "extra columns", but I wanted to be clear that the metrics are coming from the extra columns.